src/main/scala/io/iohk/ethereum/network/p2p/messages/PV63.scala

```diff
@@ -74,19 +74,24 @@ object PV63 {
     implicit class MptNodeEnc(obj: MptNode) extends RLPSerializable {
 
       def toRLPEncodable: RLPEncodeable = obj match {
```

Comment:
Should we maybe also replace this encoder with `MerklePatriciaTrie.nodeDec.encode` so as to not have the encoder duplicated?